README.md

```diff
@@ -165,23 +168,77 @@ In Device Simulator Express, you can use keyboard to interact with the device:
   - Acceleration detection
   - Temperature sensor
 
+### Useful Links
+- Tutorials and Example Code for BBC micro:bit:
+  - [MicroPython documentation](https://microbit-micropython.readthedocs.io/en/latest/)
+  - [BBC micro:bit examples on the official micro:bit website](https://microbit.org/projects/make-it-code-it/?filters=python)
+
 ### How to use
+
+#### 1. Start with the "Device Simulator Express [micro:bit]: New File" Command.
+
+1. Type in `"Device Simulator Express: [micro:bit] New File"` in the command palette (`CTRL/CMD + SHIFT + P` to open the command palette).  
+   <img alt='"New File" animation' src='assets/readmeFiles/microbit/microbit-new-file.gif'>
+2. Name and save your file somewhere, and we’re good to go!

```

Comment:
If I remember correctly, somebody shared feedback that the current README is too large. Have you consider breaking it up into more files?

Reply:
Yeah, its something I want to do too. I'll talk to the team about what's important or not. The problem is this README will be seen on the marketplace as well as on GitHub. After reading some blogs, people suggested to keep them the same because keeping them both updated properly is hard. So, the problem is having this one README be effective for both is tricky.

Reply:
I think this change improves the README quite a bit, so I wouldn't mind it staying like this considering it's also shown in the marketplace. Just wanted to make sure to bring up the feedback shared previously.
Up to you!

Reply:
Will leave for now, but something we will look back at near end of internship